### PR TITLE
Release of version v8.1.1

### PR DESCRIPTION
**Pull request by s3tezsky**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Preparation for release 8.1.1. This release patching NPM issue which occured after v8.1.0 release.
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: upgrade/UPGRADE-v8.2.0-dev.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# [Upgrade from v8.1.0 to v8.2.0-dev](https://github.com/shopsys/shopsys/compare/v8.1.0...master)
+# [Upgrade from v8.1.1 to v8.2.0-dev](https://github.com/shopsys/shopsys/compare/v8.1.1...8.1)
```
Comment (grossmannmartin) [inline upgrade/UPGRADE-v8.2.0-dev.md]: As we don't plan to release v8.2.0, I suggest to delete this file completely and links to this file. 
Please validate this idea with @sslt 

Or is necessary to change link (v8.1.1...8.1 is not right)

File: upgrade/upgrading-monorepo.md [hunk 2]
```diff
@@ -13,18 +13,20 @@ Typical upgrade sequence should be:
...
 ***Note:** During the execution of `build-demo-dev phing target`, there will be installed 3-rd party software as dependencies of Shopsys Framework by [composer](https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies) and [npm](https://docs.npmjs.com/about-the-public-npm-registry) with licenses that are described in document [Open Source License Acknowledgements and Third-Party Copyrights](https://github.com/shopsys/shopsys/blob/7.3/open-source-license-acknowledgements-and-third-party-copyrights.md)*
 
-## [From v8.1.0 to v8.2.0-dev]
+## [From v8.1.1 to v8.2.0-dev]
 
-## [From v8.0.0 to v8.1.0]
+## [From v8.1.0 to v8.1.1]
 
-## [From v8.0.0 to v8.0.1-dev]
+## [From v8.0.0 to v8.1.1]
```
Comment (grossmannmartin) [inline upgrade/upgrading-monorepo.md]: I don't think this heading is necessary

Comment (s3tezsky) [reply upgrade/upgrading-monorepo.md]: fixed

File: upgrade/UPGRADE-v8.2.0-dev.md (hunk 1 above)
Comment (s3tezsky) [reply upgrade/UPGRADE-v8.2.0-dev.md]: Removed

**Pull request merged**